notifier/event_handler/jira_event_handler/jira_event_handler.py

```diff
@@ -41,6 +41,9 @@ def process_run_completed(self, event):
     def process_operator_request_event(self, event):
         self._add_comment_event(event)
 
+    def process_etl_job_failed_event(self, event):
```

Comment:
This method needs to be added to `notifier/event_handler/noop_event_handler/noop_event_handler.py` also.

Reply:
done